src/utils/fetch-loader.ts

```diff
@@ -111,8 +112,8 @@ class FetchLoader implements Loader<LoaderContext> {
       callbacks.onTimeout(stats, context, this.response);
     }, config.timeout);
 
-    self
-      .fetch(this.request as Request)
+    Promise.resolve(this.request)
+      .then(self.fetch)
```

Comment:
```suggestion
    (isPromise(this.request)
      ? this.request.then(self.fetch)
      : self.fetch(this.request))
```

(isPromise(this.request) ? this.request.then(self.fetch) : self.fetch(this.request))

Reply:
I agree that this would be better, but I actually think the impact is minimal and it makes for better readability